sources/packages/api/src/database/entities/application-status.type.ts

```diff
@@ -43,4 +43,12 @@ export enum ApplicationStatus {
    * place on new record.
    */
   overwritten = "Overwritten",
+  /**
+   * Applications that are available.
+   */
+  available = "Available",
```

Comment:
These statuses do not belong to the entity layer. Please move them to the services or controller level, whatever makes more sense.